scipy/special/add_newdocs.py

```diff
@@ -3036,28 +3036,35 @@ def add_newdoc(place, name, doc):
 
     Gauss hypergeometric function 2F1(a, b; c; z).
 
+    \mathrm{hyp2f1}(a, b, c, z) = 1 + \sum_{k=0}^inf. \frac {{{a+k}\choose{a}} {{b+k}}\choose{b}}}{{c+k}\choose{c}} z^(k+1).
```

Comment:
Make this a complete sentence. Something like "The Gaussian hypergeometric function is defined as...". Also I wouldn't use binomial coefficients here, the Pochhammer symbol is more appropriate.